doc/nrf/doc_styleguide.rst

```diff
@@ -120,6 +202,43 @@ To link directly to a doxygen reference from RST, use the following Breathe doma
 
       :cpp:enumerator:`ENUM_VALUE <DOXYGEN_GROUP::ENUM_VALUE>`
 
+PCB names
+=========
+
+When referring to specific Printed Circuit Board elements, use capitalization.
+For example, *Button 1*, *Switch 3*, *LED 1*.
+
+If you provide the code of a specific PCB element, write it in bold: **SW3**.
+
+If you are referring to a generic PCB element, do *not* use capitalization: *button*, *switch*, *LEDs*.
+
+
+Tables
+======
+
+Follow Zephyr's :ref:`zephyr:doc_guidelines` for tables.
+However, do not add table titles to describe the table.
+Instead, make sure to introduce the table with a short sentence that describes table contents.
+For example:
+
+.. code-block:: python
+
+    The following table lists something.
+
+    .. list-table::
+        :widths: 15 20 40
+        :header-rows: 1
+
+        * - Heading 1
+          - Heading 2
+          - Heading 3
+        * - body row 1, column 1
+          - body row 1, column 2
+          - body row 1, column 3
+
+This caption guideline applies to both grid and list tables.
```

Comment:
Do we need this? There are several other tables types, and they are not forbidden to use. And the caption guideline applies to all of them.

Reply:
Removed, thanks.